frontend/src/component/commandBar/CommandRecent.tsx

```diff
@@ -57,7 +56,7 @@ export const CommandRecent = ({
             groupName='Quick suggestions'
             onClick={onClick}
         >
-            <List>{buttons}</List>
+            {buttons}
```

Comment:
This was the issue, buttons were in List here and also in subcomponent. Now it is not double wrapped.